### ISSUE TITLE
Feature request: Babel 6 with Mithril

### ISSUE THREAD

**Issue by ZaDarkSide** [capaj/jspm-mithril#6]
I would like to use the newer Babel 6 with Mithril.

**Comment by capaj** [comment]
This is possible when using latest JSPM 0.17.0

**Comment by ZaDarkSide** [comment]
Yes, I am using JSPM 0.17.0, but I can't find a working boilerplate with Babel 6, Mithril, MSX (https://github.com/insin/msx), Babel MSX Transformer (https://github.com/Naddiseo/babel-plugin-mjsx)

Right now I'm using my own boilerplate but uses RequireJS instead of SystemJS and no JSPM.
